### PR TITLE
OPENEUROPA-1326: Add needed files

### PR DESCRIPTION
**Pull request by dxvargas**
### Description

Add initial files.

### Change log

- Added: Basic files.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+workspace:
+  base: /test
+  path: drupal-core-require-dev
```
Comment (voidtek) [inline .drone.yml]: `path: drupal-core-require-dev` should be `path: drupal-core-require`

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "openeuropa/drupal-core-require",
+    "description": "Dependencies and Drupal core patches needed by OpenEuropa components.",
+    "type": "library",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "drupal/core": "^8.6"
```
Comment (voidtek) [inline composer.json]: ` "drupal/core": "^8.6" ` should be  `"drupal/core": "8.6.x-dev"`

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+services:
+  web:
+    image: fpfis/httpd-php-dev:7.1
+    environment:
+    - DOCUMENT_ROOT=/test/drupal-core-require-dev
```
Comment (voidtek) [inline .drone.yml]: `- DOCUMENT_ROOT=/test/drupal-core-require-dev` should be `- DOCUMENT_ROOT=/test/drupal-core-require`

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "openeuropa/drupal-core-require",
+    "description": "Dependencies and Drupal core patches needed by OpenEuropa components.",
+    "type": "library",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
```
Comment (voidtek) [inline composer.json]: the dependency `"cweagans/composer-patches": "^1.6"` is missing

File: .drone.yml (hunk 1 above)
Comment (dxvargas) [reply .drone.yml]: Done

File: composer.json (hunk 2 above)
Comment (dxvargas) [reply composer.json]: Done

File: .drone.yml (hunk 3 above)
Comment (dxvargas) [reply .drone.yml]: Done

File: composer.json (hunk 4 above)
Comment (dxvargas) [reply composer.json]: Done

**Pull request merged**